examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -207,6 +223,11 @@ class CastingPlayer : public std::enable_shared_from_this<CastingPlayer>
      */
     CHIP_ERROR SendUserDirectedCommissioningRequest();
 
+    /**
+     * @brief Builds an IdentificationDeclaration message to be sent to this CastingPlayer
+     */
+    chip::Protocols::UserDirectedCommissioning::IdentificationDeclaration buildIdentificationDeclarationMessage();
+
```

Comment:
This should be moved to the IdentificationDeclarationOptions class like I mentioned in my previous comment: https://github.com/project-chip/connectedhomeip/pull/33283#discussion_r1589454354